### DISCUSSION TITLE
Is this FW compatible with the Ender 3 v2 Neo

### DISCUSSION THREAD

**Discussion by lemmingDev** [mriscoc/Ender3V2S1#455] [Announcements]
Hi

I notice in the changelog it mentions thumbnail support for the neo

If it's compatible, which version should I flash?

Thanks

**Comment by mriscoc** [comment]
Sorry for the confusion, the latest release support displays the preview image generated for the Creality slicer in the Neo format. But to see the preview it is necessary to have a DWIN display.

**Comment by mriscoc** [comment]
Duplicated of https://github.com/mriscoc/Ender3V2S1/discussions/413
